web/static/web/js/dropdown.js

```diff
@@ -1,6 +1,6 @@
 $(function () {
-    $('.burger').click(function () {
-        $('.nav').toggleClass('active');
+    $('#burger').click(function () {
+        $('#header').toggleClass('active');
     });
```

Comment:
Think that you could just delete this file instead of changing it. From what I could see, it is not used anywhere in the project.